components/alert/AlertPopulations.vue

```diff
@@ -56,17 +56,47 @@
 
       <div v-show="mode === 'number'">
         <line-chart
+          v-if="!(alert.duration > 5)"
           :chart-data="dataCollection"
           :options="popNumberChartOptions"
           style="width: 100%; height: 400px"
         ></line-chart>
+        <div
+          v-else
+          style="width: 100%; height: 400px"
+          class="flex justify-center items-center flex-wrap"
+        >
+          <h1 class="w-full">
+            Awating data...
+            <font-awesome-icon
+              :icon="['fa', 'sync']"
+              class="animate-spin"
+            ></font-awesome-icon>
+          </h1>
+          <h3>Data is gathered once a minute</h3>
+        </div>
```

Comment:
```suggestion
        <div
          v-else
          style="width: 100%; height: 400px"
          class="flex justify-center items-center flex-wrap"
        >
          <div>
            <h1 class="w-full">
              Awaiting data...
              <font-awesome-icon
                :icon="['fa', 'sync']"
                class="animate-spin"
              ></font-awesome-icon>
            </h1>
            <p class="text-sm">Data is gathered once a minute</p>
          </div>
        </div>
```

Reply:
This is cheating but works, the div itself and its content should centralise horrifically and vertically

Reply:
Also never use style tags please.

Reply:
> Also never use style tags please.
I am assuming you are talking about 
`style="width: 100%; height: 400px"`
These style tags were written by you
![image](https://user-images.githubusercontent.com/61867169/183287662-be06ab96-5776-46ec-97c8-2349bbc211ed.png)

Reply:
Ah, heh maybe then correct my error and convert the width into `w-full` and leave the height, the height exists for the chart and I don't think we can get round that

Reply:
But generally it's a big nono to use inline styles, but in some cases, like this one, you can't avoid it.

Reply:
We could use h-96 which has height of 384px and looks quite similar

Reply:
Also h-fit works